lib/src/components/ComponentWrapper.tsx

```diff
@@ -1,16 +1,24 @@
-import * as React from 'react';
-import * as  _ from 'lodash';
-import * as ReactLifecyclesCompat from 'react-lifecycles-compat';
+import * as React from "react";
```

Comment:
Can you please auto formatting and keep only actual changes in the PR? Thanks 👍

Reply:
oh, sorry. Sure

Reply:
No worries, thanks :)